### PR TITLE
added sqs information

### PR DESCRIPTION
**Pull request by JKashyap96**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by JKashyap96** [comment]
We will merge into main when we are ready for release next quarter. The new version of AWSO solution will be release and these docs will be updated, hence created a different branch to store till then.

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md [hunk 1]
```diff
@@ -1463,7 +1463,7 @@ Parameters will take default values as defined under the default column.
 
 This installs the following:
 
-* Apps: AWS EC2, AWS Application Load Balancer, Amazon RDS, AWS API Gateway, AWS Lambda, AWS DynamoDB, AWS ECS, Amazon ElastiCache, AWS NLB, and AWS SNS.
+* Apps: AWS EC2, AWS Application Load Balancer, Amazon RDS, AWS API Gateway, AWS Lambda, AWS DynamoDB, AWS ECS, Amazon ElastiCache, AWS NLB, AWS SNS and AWS SQS.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md:1466]:
```suggestion
* Apps: AWS EC2, AWS Application Load Balancer, Amazon RDS, AWS API Gateway, AWS Lambda, AWS DynamoDB, AWS ECS, Amazon ElastiCache, AWS NLB, AWS SNS, and AWS SQS.
```

File: docs/observability/aws/faq.md [hunk 2]
```diff
@@ -22,11 +22,11 @@ Our AWS Observability solution enables you to view your entire AWS environment i
 
 * **AWS Observability Solution** - Sumo Logic’s AWS Observability solution pulls in data across key AWS services and accounts to give a unified view of AWS environments. Easily navigate from overview dashboards into account, region, availability zone, or service specific views. Intuitive navigation ensures teams can quickly resolve issues, minimize downtime, and improve system availability. See [About AWS Observability](docs/observability/aws/about.md).
 
-See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, and SNS.
+See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, SNS and SQS.
```
Comment (kimsauce) [inline docs/observability/aws/faq.md:25]:
```suggestion
See [AWS Observability Apps](/docs/observability/aws/integrations) for detailed information on all supported apps including Application, Classic, and Network Load Balancer, DynamoDB, EC2, RDS, API Gateway, ECS, ElastiCache, Lambda, SNS, and SQS.
```

File: docs/observability/aws/faq.md [hunk 3]
```diff
@@ -22,11 +22,11 @@ Our AWS Observability solution enables you to view your entire AWS environment i
...
 
 ### What AWS services are supported?
 
-Sumo Logic AWS Observability supports the following AWS services: EC2, ECS, RDS, ElastiCache, API Gateway, Lambda, DynamoDB, Application ELB, Classic ELB, Amazon SNS, and Network ELB.
+Sumo Logic AWS Observability supports the following AWS services: EC2, ECS, RDS, ElastiCache, API Gateway, Lambda, DynamoDB, Application ELB, Classic ELB, Amazon SNS, Amazon SQS and Network ELB.
```
Comment (kimsauce) [inline docs/observability/aws/faq.md:29]:
```suggestion
Sumo Logic AWS Observability supports the following AWS services: EC2, ECS, RDS, ElastiCache, API Gateway, Lambda, DynamoDB, Application ELB, Classic ELB, Amazon SNS, Amazon SQS, and Network ELB.
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
+---
+id: amazon-sqs
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:2]: Hi @JKashyap96 , is [this](https://help.sumologic.com/docs/integrations/amazon-aws/sqs/) the same app?

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+         },
+         "responseElements":null,
+         "requestID":"635ae9dd-83cc-5b42-890e-b273c168cb35",
+         "eventID":"874213f3-d852-481d-bfd1-677c20f97427",
+         "readOnly":false,
+         "eventType":"AwsApiCall",
+         "managementEvent":true,
+         "recipientAccountId":"123456789033",
+         "eventCategory":"Management",
+         "sessionCredentialFromConsole":"true"
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:62]: Fix JSON indentation

```suggestion
{
	"eventVersion":"1.08",
	"userIdentity":{
		"type":"IAMUser",
		"principalId":"HN6WE52ULYKOOQWSKPD41",
		"arn":"arn:aws:iam::123456789033:user/ron_di",
		"accountId":"123456789033",
		"accessKeyId":"1AMZ29JHKLK97M7QVQSK",
		"userName":"ron_di",
		"sessionContext":{
			"sessionIssuer":{
				
			},
			"webIdFederationData":{
				
			},
			"attributes":{
				"creationDate":"2022-10-19T09:26:18+0000",
				"mfaAuthenticated":"false"
			}
		}
	},
	"eventTime":"2022-10-19T09:26:18+0000",
	"eventSource":"sqs.amazonaws.com",
	"eventName":"SetQueueAttributes",
	"awsRegion":"us-east-1",
	"sourceIPAddress":"195.186.216.125",
	"userAgent":"AWS Internal",
	"requestParameters":{
		"attributes":{
			"Policy":"{\"Version\":\"2008-10-17\",\"Id\":\"__default_policy_ID\",\"Statement\":[{\"Sid\":\"__owner_statement\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"arn:aws:iam::956882708938:root\"},\"Action\":\"SQS:*\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\"},{\"Sid\":\"topic-subscription-arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\",\"Effect\":\"Allow\",\"Principal\":{\"AWS\":\"*\"},\"Action\":\"SQS:SendMessage\",\"Resource\":\"arn:aws:sqs:us-east-1:956882708938:JayanatTest4\",\"Condition\":{\"ArnLike\":{\"aws:SourceArn\":\"arn:aws:sns:us-east-1:956882708938:SNSAppSomya01\"}}}]}"
		},
		"queueUrl":"https://sqs.us-east-1.amazonaws.com/123456789033/pull_private_submodule_jobs.fifo"
	},
	"responseElements":null,
	"requestID":"635ae9dd-83cc-5b42-890e-b273c168cb35",
	"eventID":"874213f3-d852-481d-bfd1-677c20f97427",
	"readOnly":false,
	"eventType":"AwsApiCall",
	"managementEvent":true,
	"recipientAccountId":"123456789033",
	"eventCategory":"Management",
	"sessionCredentialFromConsole":"true"
}
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+### Sample Query
+
+**Messages Received (Metric based)**:
+
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
+
+**Top 10 users (CloudTrail Log based)**:
+
+```
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:76]:
```suggestion
```sql
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
+
+* **Rule Name**: AwsObservabilitySQSCloudTrailLogsFER
+* **Applied at**: Ingest Time
+* **Scope (Specific Data)**: account=* eventname eventsource "sqs.amazonaws.com"
+* **Parse Expression**:
+
+```
+json "userIdentity", "eventSource", "eventName", "awsRegion", "recipientAccountId", "requestParameters", "responseElements", "sourceIPAddress" as userIdentity, event_source, event_name, region, recipient_account_id, requestParameters, responseElements, src_ip  nodrop
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:106]: Is this code snippet supposed to start with `json`?

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+
+### Audit Events
+
+The **2. Amazon SQS - Audit Events** dashboard provides the details of SQS from CloudTrail audit logs including the top users, event locations, event status associated with queues. The dashboard has panels regarding successful and failure event locations, error code
+Use this dashboard to:
+* Monitor events by status, type, queues, location and users.
+* Monitor successful, failure event locations and trends.
+* Monitor event details by users.
+* Monitor successful and error event details.
+![2.Amazon SQS Audit Events](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Amazon-SQS/AmazonSQS-AuditEvents.png)
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:156]:
```suggestion

![2.Amazon SQS Audit Events](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Amazon-SQS/AmazonSQS-AuditEvents.png)
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+### Threat Intel
+
+The **3. Amazon SQS - Threat Intel** dashboard provides insights into incoming requests to your AWS SQS services from malicious sources determined via Sumo Logic’s Threat Intel feature. Panels show detailed information on malicious IPs and the malicious confidence of each threat.
+Use this dashboard to:
+* Monitor details of threat locations and count.
+* Get details of threats by malicious confidence and malicious IPs.
+* Get details of all threats by IPs.
+![3.Amazon SQS - Threat Intel](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Amazon-SQS/AmazonSQS-ThreatIntel.png)
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:165]:
```suggestion

![3.Amazon SQS - Threat Intel](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Amazon-SQS/AmazonSQS-ThreatIntel.png)
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+}
+```
+
+
+
+### Sample Query
+
+**Messages Received (Metric based)**:
+
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:72]:
```suggestion
`metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename`
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Changes requested

File: docs/observability/aws/integrations/amazon-sqs.md (hunk 4 above)
Comment (JKashyap96) [reply docs/observability/aws/integrations/amazon-sqs.md:2]: HI @kimsauce ! This is the same app but we maintain a different page here as a few sections are not included in this page from the one you mentioned.

File: docs/observability/aws/integrations/amazon-sqs.md (hunk 7 above)
Comment (JKashyap96) [reply docs/observability/aws/integrations/amazon-sqs.md:106]: I am adding a pipe(|) here.

File: docs/integrations/amazon-aws/sqs.md [hunk 11]
```diff
@@ -199,10 +200,11 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
 The Sumo Logic App for Amazon SQS provides operational insights into your Amazon SQS utilization. The App’s preconfigured dashboards help you monitor the key metrics, view the SQS events for queue activities, and help you plan the capacity of your SQS service utilization.
 
 :::note
-We highly recommend you view these dashboards in the [Explore View](https://help.sumologic.com/Beta/AWS_Observability_Solution/07_View_AWS_Observability_Solution_Dashboards) of  the AWS Observability solution.
+We highly recommend you view these dashboards in the [Explore View](https://help.sumologic.com/Beta/AWS_Observability_Solution/07_View_AWS_Observability_Solution_Dashboards) of the AWS Observability solution.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:203]: @JKashyap96 link to new doc (this is the legacy link)

File: docs/integrations/amazon-aws/sqs.md [hunk 12]
```diff
@@ -72,8 +72,8 @@ metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* que
 
 **Top 10 users (CloudTrail Log based)**:
 
-account=* region=* namespace=aws/sqs eventname eventsource "sqs.amazonaws.com"
 ```
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:75]:
```suggestion
```sql
```

File: docs/observability/aws/faq.md (hunk 3 above)
Comment (kimsauce) [inline docs/observability/aws/faq.md:29]:
```suggestion
Sumo Logic AWS Observability supports the following AWS services: EC2, ECS, RDS, ElastiCache, API Gateway, Lambda, DynamoDB, Application ELB, Classic ELB, Amazon SNS, Amazon SQS, and Network ELB.
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 13]
```diff
@@ -0,0 +1,167 @@
...
+}
+```
+
+
+
+### Sample Query
+
+**Messages Received (Metric based)**:
+
+`metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename`
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:72]: needs triple backticks, as this is a code snippet

```suggestion
`metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename`
```

File: docs/observability/aws/integrations/amazon-sqs.md [hunk 14]
```diff
@@ -0,0 +1,167 @@
...
+## Field Extraction Rule(s)
+
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
+
+* **Rule Name**: AwsObservabilitySQSCloudTrailLogsFER
+* **Applied at**: Ingest Time
+* **Scope (Specific Data)**: account=* eventname eventsource "sqs.amazonaws.com"
+* **Parse Expression**:
+
+```
```
Comment (kimsauce) [inline docs/observability/aws/integrations/amazon-sqs.md:105]:
```suggestion
```sql
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Please see my comments/fixes

File: docs/integrations/amazon-aws/sqs.md (hunk 11 above)
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:203]:
```suggestion
We highly recommend you view these dashboards in the [Explore View](/docs/dashboards-new/explore-view) of the AWS Observability solution.
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**